lib/sobelow/traversal/send_file.ex

```diff
@@ -13,7 +13,7 @@ defmodule Sobelow.Traversal.SendFile do
   end
 
   ## send_file(conn, status, file, offset \\ 0, length \\ :all)
-  defp parse_def(fun) do
+  def parse_def(fun) do
```

Comment:
I'm not sure that we want to change the scope of this function in order to test it. I only did so to match the pattern for the tests for `Sobelow.XSS.ContentType`. I'm happy to rewrite this PR in a different way so feel free to suggest something else :smile:

Reply:
This is fine! Thanks for the PRs :)